assets/app/view/game/route_selector.rb

```diff
@@ -270,6 +271,52 @@ def actions(render_halts)
           h(:button, { style: submit_style, on: { click: submit } }, 'Submit ' + revenue + subsidy),
         ])
       end
+
+      def dividend_chart
```

Comment:
i thought had made one of these before or am i misremembering

Reply:
I have something very similar I added to the dividend step for 18Mag.